pkg/catalog/catalog_test.go

```diff
@@ -140,7 +186,7 @@ func TestList(t *testing.T) {
 			// Fake the client with the appropriate arguments
 			client, fakeClientSet := occlient.FakeNew()
 			fakeClientSet.ImageClientset.PrependReactor("list", "imagestreams", func(action ktesting.Action) (bool, runtime.Object, error) {
-				return true, testingutil.FakeImageStreams(tt.args.name, tt.args.namespace, tt.args.tags), nil
+				return true, testingutil.FakeImageStreams(tt.args.name, tt.args.namespace, tt.args.tags, tt.args.currentNSImageName, tt.args.currentNamespace, tt.args.currentNSImageTags), nil
```

Comment:
why not just call ` FakeImageStreams` twice? Once to add images to "openshift" namespace, second time to add  them to the other namespace. You don't have to modify ` FakeImageStreams` function at all.

Reply:
will try this
but how would this work? the prependReactor doesn't take params as arg right to expect it to return 2 different NS values for 2 different calls based on params passed?

Reply:
@kadel Hey but it worked